styles/base.less

```diff
@@ -5,7 +5,7 @@
 @import "ui-variables";
 @import "colors";
 
-.atom-text-editor, :host {
+.atom-text-editor, .atom-text-editor {
```

Comment:
Okay no worries! I got it all cleaned up. Thanks again for doing all the heavy lifting on this one!